eq-author/src/App/settings/ThemesPage.js

```diff
@@ -137,32 +210,40 @@ const ThemesPage = ({ questionnaire }) => {
                       />
                     </Field>
                     <HorizontalSeparator />
-                    {themes.map(({ title, defaultOpen }) => (
-                      <CollapsibleToggled
-                        key={`${title}-toggle`}
-                        title={title}
-                        defaultOpen={defaultOpen}
-                      >
-                        {/* Added some filler text to demonstrate the opening and 
+                    {matchThemes(themes, questionnaireThemes).map(
+                      ({ shortName, title, eqId, enabled }) => (
+                        <CollapsibleToggled
+                          key={`${title}-toggle`}
+                          title={title}
+                          defaultOpen={enabled}
```

Comment:
```suggestion
                          isOpen={enabled}
```

Now that the open and close state is controlled outside of the component, it might make more sense to rename this prop to make it clearer that it is what controls the open and close state at all times, not just a default for the initial render.